config/config.go

```diff
@@ -259,6 +261,8 @@ func (c *Config) LoadConfigFile(path string) error {
 func (c *Config) LoadConfig(in []byte) error {
 	err := yaml.Unmarshal(in, c)
 	if err != nil {
+		log.Println(err)
+		log.Println("this should be cool", yaml.FormatError(err, true, true))
```

Comment:
Using `yaml.FormatError` is a very good idea. But could you exclude it from this Pull Request?

Reply:
Sorry, that was a leftover from debug :grin:, removing.

By the way, the last version of github.com/goccy/go-yaml has a more informative error message in case of invalid yaml.
Do you mind if I update it in go.mod?